sklearn/linear_model/logistic.py

```diff
@@ -1607,6 +1658,9 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         Array of C i.e. inverse of regularization parameter values used
         for cross-validation.
 
+    l1_ratios_ : array
+        Array of l1_ratios used for cross-validation.
+
     coefs_paths_ : array, shape ``(n_folds, len(Cs_), n_features)`` or \
```

Comment:
Shouldn't there be one for each `l1_ratio`? Surely l1_ratios should become another dimension. See also shape of `ElasticNetCV.mse_paths_` (which is quite different... and I'm not really sure why we're storing all the coefs; it seems expensive)

Reply:
Similarly, the shape mentioned in `scores_` below shoulld be updated.

Reply:
I don't understand what you mean. `l1_ratios_` is just the 'processed' version of the `l1_ratios` parameter, so it's just an array of floats. Just like the `Cs_` attribute right above.

Reply:
But there should be a distinct vector of coefficients for each C and l1_ratio so the shapes documented here should be updated

Reply:
Yes absolutely, sorry I totally misread your comment.
The new shape is ``(n_folds, len(Cs_) x len(l1_ratios), n_features)``, I assume this is more correct as adding a dimension could break users code?